src/content/pages/build/taco.md

```diff
@@ -0,0 +1,213 @@
+  image: /images/taco-grid.svg
+  background: /images/glowing-background.png
+tacoDiagrams:
+  - title: "Generating a DKG Cohort"
+    preTitle: "learn"
+    description: "Independent nodes sampled from the Threshold network form a cohort – who collectively generate a DKG public key."
+    image: /images/first-taco-diagram.svg
+    buttons:
+      - label: "nucypher/ferveo"
+        url: https://github.com/nucypher/ferveo
+        variant: EXTERNAL_OUTLINE
+      - label: "nucypher-contracts"
+        url: https://github.com/nucypher/nucypher-contracts
+        variant: EXTERNAL_OUTLINE
+  - title: "Encrypting Private Data"
+    preTitle: "learn"
+    description: "Data owners specify conditions for decryption, embedded with private data and encrypted with the DKG public key."
+    image: "/images/second-taco-diagram.svg"
+    buttons:
+      - label: "taco-web"
+        url: https://github.com/nucypher/taco-web
+        variant: EXTERNAL_OUTLINE
+      - label: "nucypher/main"
+        url: https://github.com/nucypher/nucypher/tree/main
+        variant: EXTERNAL_OUTLINE
+  - title: "Accessing Decrypted Data"
+    preTitle: "learn"
+    description: "Requesters present state to the cohort, who verify conditions fulfillment before provisioning decryption material."
+    image: "/images/third-taco-diagram.svg"
+    buttons:
+      - label: "taco-web"
+        url: https://github.com/nucypher/taco-web
+        variant: EXTERNAL_OUTLINE
+      - label: "nucypher/main"
+        url: https://github.com/nucypher/nucypher/tree/main
+        variant: EXTERNAL_OUTLINE
+tacoExamplesInfo:
+  title: "Access conditions that fit your use case"
+  preTitle: "build"
+  description: "Whatever the business logic for data access, TACo nodes will enforce it. Specify and combine EVM, RPC and time-based conditions. Non-blockchain coming soon."
+  image: "/images/ecosystem-grid.svg"
+  rowReverse: false
+tacoExamples:
+  - title: "NFT-gating"
+    description: "Predicate decryption rights on the holding of special-purpose NFTs. Works for digital media, event streams and beyond."
+    image: "/images/nft-gating.png"
+  - title: "Secret recovery"
+    description: "Share your seed phrase or password  – to beneficiaries or your future self – without risking DOS by an intermediary."
+    image: "/images/secret-recovery.png"
+  - title: "Inference protection"
+    description: "Prevent third-parties from harvesting or blocking communication between end-users and generative models."
+    image: "/images/inference-protection.png"
+tacoTableInfo:
+  title: "The Zero Compromise Solution"
+  highlighted: ["Zero", "Compromise"]
+  description: "Choose a service that’s easy to integrate and that’s actually decentralized."
+  rowReverse: false
+tacoTable:
+  headers:
+    [
+      "Feature / Service Choice",
+      "TACo",
+      "LIT",
+      "EVM Contract",
+      "PKI",
+      "Cloud Service",
+    ]
+  features:
+    - label: "Scalable to millions of users"
```

Comment:
The left hand column text (Scalable to millions of users, Arbitrary access conditons, etc) should be aligned left rather than centered

Reply:
Addressed in https://github.com/threshold-network/website/pull/113/commits/eb14689ece7a466e8441182e7c6e7206013c3e55